src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -1,373 +1,58 @@
-            <template v-else-if="filing.isFutureEffectiveIa || filing.isFutureEffectiveAlteration ||
-              filing.isFutureEffectiveDissolution"
-            >
-              <FutureEffective :filing=filing class="mt-6" />
-            </template>
-
-            <template v-else-if="filing.isTypeLimitedRestoration">
-              <LimitedRestorationFiling :filing="filing" class="mt-6" />
-            </template>
-
-            <!-- is this a generic paid (not yet completed) filing? -->
-            <template v-else-if="EnumUtilities.isStatusPaid(filing)">
-              <PendingFiling :filing=filing class="mt-6" />
-            </template>
-
-            <!-- is this a paper filing? -->
-            <template v-else-if="filing.availableOnPaperOnly">
-              <PaperFiling class="mt-6" />
-            </template>
-
-            <!-- is this a consent to continuation out filing? -->
-            <template v-else-if="EnumUtilities.isTypeConsentContinuationOut(filing)">
-              <p class="mt-4">
-                This consent is valid <strong>until {{filing.expiry}} at 12:01 am Pacific time</strong>.
-              </p>
-              <p class="mt-4">{{filing.comment}}</p>
-              <p v-if="filing.fileNumber" class="mt-4 mb-0">Court Order Number: {{filing.fileNumber}}</p>
-              <p v-if="filing.planOfArrangement" class="mt-0">{{filing.planOfArrangement}}</p>
-            </template>
-
-            <!-- else this must be a completed filing -->
-            <template v-else>
-              <!-- no details -->
-            </template>
-
-            <!-- the documents section -->
-            <template v-if="!EnumUtilities.isTypeCourtOrder(filing) && filing.documents &&
-              filing.documents.length > 0"
-            >
-              <v-divider class="my-6" />
-              <DocumentsList
-                :filing=filing
-                :loadingOne=loadingOne
-                :loadingAll=loadingAll
-                :loadingOneIndex=loadingOneIndex
-                @downloadOne="downloadOne(...arguments)"
-                @downloadAll="downloadAll($event)"
-              />
-            </template>
-
-            <!-- the details (comments) section -->
-            <template v-if="filing.comments && filing.commentsCount > 0">
-              <v-divider class="my-6" />
-              <DetailsList
-                :filing=filing
-                @showCommentDialog="showCommentDialog($event)"
-              />
-            </template>
-
-          </v-expansion-panel-content>
-        </v-expansion-panel>
+        />
       </v-expansion-panels>
     </div>
 
     <!-- No Results Message -->
     <v-card class="no-results" flat v-if="!showHistoryPanel">
       <v-card-text>
-        <template v-if="!!tempRegNumber">
+        <template v-if="isTemporaryRegistration">
           <div class="no-results__subtitle">Complete your filing to display</div>
         </template>
```

Comment:
I think it's only a few lines and not "unclean" enough to make me want another component.

What changing this would do is move this logic (isTemporaryRegistration vs isBusiness) to the parent component (Dashboard), so you wouldn't be getting rid of any code.

I can do it if you insist, but I don't see a strong reason to do it.